src/renderer/components/SyncView/SyncAppBar.js

```diff
@@ -7,15 +7,37 @@ import { makeStyles } from '@material-ui/core/styles'
 import { Wifi, WifiOff } from '@material-ui/icons'
 import { defineMessages, useIntl } from 'react-intl'
 import wifi from 'node-wifi'
+import { Tooltip } from '@material-ui/core'
 
 wifi.init({
   iface: null
 })
 
+const getQualityStyle = (connection) => {
+  if (connection.quality < 30) return { color: 'white', backgroundColor: 'red' }
+  if (connection.quality < 60) return { color: 'white', backgroundColor: 'orange' }
+  return { backgroundColor: '#E0E0E0' }
+}
+
+/** @returns {keyof m} */
+const getWifiConnectionMessage = (connection) => {
```

Comment:
Just a point of style - fine to ignore - but this might make more sense to return the message object e.g. `m.qualityTooltipPoor`, then line #67 can be `t(getWifiConnectionMessage(currentConnection))`.

Reply:
Makes sense! Then I can get rid of the JSDoc comment as well (it lived a short but intense life)